app/models/ems_refresh.rb

```diff
@@ -168,9 +168,8 @@ def self.queue_merge(targets, ems, create_task = false)
   end
 
   def self.create_refresh_task(ems, targets)
-    targets = targets.collect { |target_class, target_id| [target_class.demodulize, target_id] }
     task_options = {
-      :action => "EmsRefresh(#{ems.name}) [#{targets}]".truncate(255),
+      :action => "EmsRefresh(#{ems.name})#{targets.present? ? " #{targets.count} #{targets.first.first.to_s.split(':').last}" : ""}",
```

Comment:
> targets.first.first.to_s.split(':').last

I think this is an indication that we're on the wrong track :/

Reply:
```ruby
  target_class = targets.first&.first&.demodulize # first, class, simplified
  task_options = {
      :action => "EmsRefresh(#{ems.name})#{targets.present? ? " #{targets.count} #{target_class}" : ""}",
```

Reply:
I had also wanted to remove the target if the targets were the ems, but for some reason the code got too complicated and thought this was good enough.

Reply:
I don't think the message needs to list every VM that is being refreshed.
The customer won't know the VM id.
Maybe just "Manual Refresh" is good enough without the particular id?